### ISSUE TITLE
Should Error, Array, and Object be EmberError, EmberArray, and EmberObject?

### ISSUE THREAD

**Issue by toddjordan** [ember-cli/ember-rfc176-data#42]
In API docs we call the ember version Error, Object, and Array EmberError, EmberObject, and EmberArray.

Part of api docs relies on `localName` so we can match mapping information and give inline examples.  Is it possible to change these names?

See https://github.com/ember-learn/ember-api-docs/issues/423

**Comment by rwjblue** [comment]
Do you mean the names that we use for the imported function/class/etc?

**Comment by toddjordan** [comment]
yeah, in apidocs we use the `localName` for x in `import x from 'y';`

**Comment by Turbo87** [comment]
@toddjordan that is what https://github.com/ember-cli/ember-rfc176-data/blob/master/reserved.json is for. If `localName` is part of `reserved.json` than it should be prefixed with `Ember`. At least I think that was the logic that was/is encoded in the original codemod.

**Comment by toddjordan** [comment]
@Turbo87 ok, then are you suggesting that we should update mappings.json to make the reserved names prefixed with Ember, or are you suggesting we consult reserved.json in apidocs and programatically prefix them?

**Comment by Turbo87** [comment]
I think there was probably a reason why localName was not prefixed, but I'm not sure what it was. If we want to change it we should check that it does not break anything else.
